### DISCUSSION TITLE
PR Template: What should be on there?

### DISCUSSION THREAD

**Discussion by Thomas-Boi** [devicons/devicon#526] [Ideas]
Hi all,

@Panquesito7 has a great idea of tagging branches with icons with the `feature:icon` label (see #523). After some discussion, we thought that this is a good chance to create a PR Template, specifically those that add new icons to the repo.  This discussion is to discuss what should be on the template.

Here are some details that I think we will need:
- Title must match format of "new icon: [icon name] ([different versions])". I'm not sure whether the template can do this but if it can, this will be great.
  - This is required because the `peek:icon` workflow uses regexp to find the icons that were added in this PR. I'll change this in the future so it uses an action instead.
- Body:
  - Link to the technology's official page, Wiki page, or GitHub repo.
  - Notes or important info about the logo if required. Things like "this is an older icon but is still used" or "the wordmark version is my own work and is unofficial" etc...

I don't think contributors need to post images of the svgs in the PR since us maintainers will have to double check the file changes and see the logos anyway.

What do you guys think 🤔?

**Comment by Panquesito7** [comment]
> * I'm not sure whether the template can do this

Yep, templates can do this. You can specify a title and then tell the user to modify it.

> * Body:

This looks good to me. 👍 

> I don't think contributors need to post images of the SVGs in the PR

I'm actually neutral about this. Maintainers can check out the files directly, but IMHO it would be easier to see them in the first comment in the PR. 🙂
